backend/backend/services/business_logic.py

```diff
@@ -76,3 +76,11 @@ async def add_github_comment(context: Context, repo: str, pr_number: int,
     response = await agent.ainvoke(repo, pr_number, request)
     logger.info(f"GitHub comment response: {response}")
     return response
+
+
+async def review_github_pr(context: Context, repo: str, pr_number: int):
```

Comment:
The function `review_github_pr` has been modified to use a different agent. This may affect its functionality.